docs/blog/posts/2024/week33.md

```diff
@@ -0,0 +1,81 @@
+---
+title: '@warp-ds v2 - Stable releases'
+date: 2024-08-15
+---
+
+Stable releases with DBA changes for web are now available.
+
+---
+
+# How to Update Your Web Apps for DBA
+
+### Packages to Install from NPM
+
+| Package Name        | Version        |
+|---------------------|----------------|
+| `@warp-ds/css`      | `2.0.0`        |
+| `@warp-ds/uno`      | `2.0.0`        |
+| `@warp-ds/react`    | `2.0.0`        |
+| `@warp-ds/vue`      | `2.1.0`        |
+| `@warp-ds/elements` | `2.0.0`        |
+| `@warp-ds/icons`    | `2.1.0-next.2` |
+
+If you have already followed the [previous instructions for the DBA beta changes](https://warp-ds.github.io/tech-docs/blog/posts/2024/warp-2-0) and have disabled EIK import mapping for `react`, `vue` & `elements`, you can now re-enable it.
+
+For further instructions on how to migrate from v1 to v2, please follow [these instructions](https://warp-ds.github.io/tech-docs/blog/posts/2024/warp-2-0#migrating-from-v1-to-v2).
+
+## Breaking changes
+### @warp-ds/css 2.0.0
```

Comment:
I'm thinking that the css breaking changes are the least relevant to users, so perhaps we could move this section to the bottom?